.prospector.yaml

```diff
@@ -0,0 +1,47 @@
+inherits:
+  - strictness_medium
```

Comment:
what does this refer to?

Reply:
This just sets the baseline level of linting messages to suppress. 
If it was set to low I likely wouldn't need to turn off as many specific errors checks but it would also have turned off things we want to check for.
medium is the default, but I just put it here for visibility. 
https://prospector.landscape.io/en/master/profiles.html#builtin-profiles